app/src/main/java/app/revanced/integrations/patches/GeneralAdsPatch.java

```diff
@@ -61,7 +61,9 @@ public GeneralAdsPatch() {
                 "hero_promo_image",
                 "statement_banner",
                 "carousel_footered_layout",
-                "text_image_button_layout"
+                "text_image_button_layout",
+                "primetime_promo",
+                "feature_grid_interstitial"
```

Comment:
Can you please explain what either of those blocks?

Reply:
https://github.com/revanced/revanced-patches/pull/1781